README.md

```diff
@@ -12,29 +12,30 @@ The documents are written in RST and are built using [Sphinx](http://www.sphinx-
 - Every build will generate docs for all versions maintained.
 - The root documentation points to the latest released verion. Navigate to `master` version for the development version of this documentation.
 
+#### Pre-requisites
+
+* Python 3
+* Nodejs
+* NPM
+* pip3 install -U Sphinx
 
 ## Building the documentation
 
 There are two formats of documentation built for every version.
+- `pip3 install -r requirements.txt` to install the pre-requisites for docs.gauge.org
 
-- `make zip` will build `html` and zipped `singlehtml` documentation. The output is available in `_build/html` directory.
+- `make local-build` will build `html` and zipped `singlehtml` documentation. The output is available in `_build` directory.
```

Comment:
I do not think any zip (`singlehtml`) is created, please verify that.

Reply:
Yes, singlehtml is no longer created, removing that.